packages/application-shell/src/components/project-switcher/project-switcher.tsx

```diff
@@ -28,10 +29,22 @@ type Props = {
 type OptionType = Pick<TProject, 'key' | 'name' | 'suspension' | 'expiry'> & {
   label: string;
 };
-type CustomValueContainerProps = ValueContainerProps<OptionType, false> & {
+type CustomValueContainerProps = ValueContainerProps & {
   projectCount: number;
 };
 
+// FIXME: properly expose these static component types from uikit.
+type SelectInputWithStaticComponents = {
+  ValueContainer: typeof components.ValueContainer;
+  Option: typeof components.Option;
+};
```

Comment:
This refers to this part, which is currently not really typed

https://github.com/commercetools/ui-kit/blob/edf45ad0196138318050e63d84938abf5ae11aca/packages/components/inputs/select-input/src/select-input.tsx#L515-L519

Reply:
FYI: I opened a PR to fix this https://github.com/commercetools/ui-kit/pull/2140